chain/chain/src/metrics.rs

```diff
@@ -271,3 +273,172 @@ pub static SAVED_LATEST_WITNESSES_SIZE: Lazy<IntGauge> = Lazy::new(|| {
     )
     .unwrap()
 });
+
```

Comment:
let's move this to a separate module to avoid having large files, something like `chain/chain/src/stateless_validation/metrics.rs`